### PR TITLE
Add kanagram manifest (from invent.kde.org/packaging/flatpak-kde-appl…

### PR DESCRIPTION
**Pull request by jpwhiting**
…ications.git)

I'm not sure if this is the kanagram "manifest" or if manifest is something else entirely. (The docs at https://github.com/flathub/flathub/wiki/App-Submission didn't make that clear and I also don't see any other app manifests in this repository at all somehow...)

### CODE REVIEW

File: org.kde.kanagram.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kanagram",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "tags": ["nightly"],
```
Comment (tsdgeos) [inline org.kde.kanagram.json:8]: not a nightly, please remove

File: org.kde.kanagram.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": ["--share=network", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
+
+    "modules": [
+        {
+            "name": "libkeduvocdocument",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://anongit.kde.org/libkeduvocdocument.git" } ]
```
Comment (tsdgeos) [inline org.kde.kanagram.json:16]: We build from stable tarbllas, not from git, please fix this. See https://github.com/flathub/org.kde.kgeography/blob/master/org.kde.kgeography.json for example

**Comment by jpwhiting** [comment]
Cool thanks, fixed, built locally, installed and tested, seems to work ok.

**Comment by tsdgeos** [comment]
bot build org.kde.kanagram

File: org.kde.kanagram.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.kanagram",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "desktop-file-name-suffix": " (Nightly)",
```
Comment (tsdgeos) [inline org.kde.kanagram.json:8]: it still says nightly here

**Comment by tsdgeos** [comment]
bot, build org.kde.kanagram

File: org.kde.kanagram.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.kanagram",
+    "branch": "master",
```
Comment (tsdgeos) [inline org.kde.kanagram.json:3]: and the branch is not needed either

**Comment by tsdgeos** [comment]
We need a oars path to make the checker happy

Please see https://github.com/flathub/org.kde.kcalc/

And make sure you merge that upstream too so we can remove it on next release

**Comment by jpwhiting** [comment]
bot build org.kde.kanagram

**Comment by tsdgeos** [comment]
you missed the comma :D

**Comment by tsdgeos** [comment]
bot, build org.kde.kanagram

**Comment by jpwhiting** [comment]
Doh! thanks.

File: org.kde.kanagram.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{
+    "id": "org.kde.kanagram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "finish-args": ["--share=network", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
```
Comment (tsdgeos) [inline org.kde.kanagram.json:7]: I think we need       "--device=dri",
to get away from that ugly warning when starting

libGL error: MESA-LOADER: failed to retrieve device information
libGL error: Version 4 or later of flush extension not found
libGL error: failed to load driver: i915
libGL error: failed to open /dev/dri/card0: El fitxer o directori no existeix
libGL error: failed to load driver: i965

**Comment by tsdgeos** [comment]
bot, build org.kde.kanagram

**Comment by tsdgeos** [comment]
Looks reasonable to me :)

File: org.kde.kanagram.json [hunk 6]
```diff
@@ -0,0 +1,49 @@
+{
+    "id": "org.kde.kanagram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "rename-icon": "kanagram",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "finish-args": ["--share=network", "--device=dri", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
```
Comment (barthalion) [inline org.kde.kanagram.json:8]: It should use `fallback-x11` instead of `x11` if it works natively on Wayland.

File: org.kde.kanagram.json [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (barthalion) [inline org.kde.kanagram.json:9]: You should add `cleanup` array with at least `/include` and `/lib/cmake` to save some space.

**Comment by jpwhiting** [comment]
bot, build org.kde.kanagram

**Pull request closed without merging**